mirrord/cli/src/operator.rs

```diff
@@ -58,6 +64,91 @@ async fn operator_setup(
     Ok(())
 }
 
+async fn operator_status() -> Result<()> {
+    let progress = TaskProgress::new("Operator Status").fail_on_drop(true);
+
+    let status_api: Api<MirrordOperatorCrd> = Api::all(
+        create_kube_api(None)
+            .await
+            .map_err(CliError::KubernetesApiFailed)?,
+    );
+
+    let status_progress = progress.subtask("Fetching Status");
+
+    let mirrord_status = match status_api
+        .get("operator-controller")
+        .await
+        .map_err(KubeApiError::KubeError)
+        .map_err(OperatorApiError::KubeApiError)
+        .map_err(CliError::OperatorConnectionFailed)
+    {
+        Ok(status) => status,
+        Err(err) => {
+            status_progress.fail_with("Unable to get Status");
+
+            return Err(err);
+        }
+    };
+
+    let mut sessions = Table::new();
+
+    sessions.add_row(row!["Target", "User", "Session Duration"]);
+
+    status_progress
+        .subtask(&format!(
+            "Operator version: {}",
+            mirrord_status.spec.operator_version
+        ))
+        .done();
+
+    status_progress
+        .subtask(&format!(
+            "Operator default namespace: {}",
+            mirrord_status.spec.default_namespace
+        ))
+        .done();
+
+    let license_progress = status_progress.subtask("Operator License");
+
+    license_progress
```

Comment:
Are you using the progress reporter to print out the output of the function? It's confusing (+ can be disabled by the user), why not use a normal print?